sample-compose/app/build.gradle.kts

```diff
@@ -158,6 +162,13 @@ dependencies {
     testImplementation("io.mockk:mockk:${Versions.TEST_MOCKK_VERSION}")
     testImplementation("app.cash.turbine:turbine:${Versions.TEST_TURBINE}")
 
+    // Instrument test with Robolectric
+    // Need to have BOM for testImplementation https://github.com/gradle/gradle/issues/23347
+    testImplementation(platform("androidx.compose:compose-bom:${Versions.COMPOSE_BOM_VERSION}"))
+    testImplementation("androidx.compose.ui:ui-test-junit4")
+    testImplementation("androidx.test:rules:${Versions.TEST_RULES_VERSION}")
+    testImplementation("org.robolectric:robolectric:${Versions.TEST_ROBOLECTRIC_VERSION}")
+
     // Instrument test
```

Comment:
The word `Instrument` could make us confuse when running UI tests with Robolectric (as unit tests). I will replace `Instrument` with `UI` in the comment.